### ISSUE TITLE
Higher Bet Amounts For Low Risk Predictions

### ISSUE THREAD

**Issue by Jaivantv** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#232]
I found that while using the miner there were some predictions with an obvious winner or predictions with choices having a very large difference in percentage i.e. low risk predictions where its easy to make a few easy channel points. But with the odds being close to 1 it meant a very minimal return unless the bet amount was large and so I thought of an idea where the miner would place bets according to a higher percentage of balance as the difference between the percentage of odds increases. I implemented this as code suitable only for my preferences by altering a few lines in [Bet.py](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/blob/master/TwitchChannelPointsMiner/classes/entities/Bet.py)

https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/blob/40dcf78a7d9720b75c4c16810f122d36bb94e1d0/TwitchChannelPointsMiner/classes/entities/Bet.py#L256-L259

Altered Code:
```
            difference = abs(
                self.outcomes[0][OutcomeKeys.ODDS_PERCENTAGE]
                - self.outcomes[1][OutcomeKeys.ODDS_PERCENTAGE]
            )
            if (
                difference>49
                and difference<81
            ):
                self.decision["amount"] = min(
                int(balance * 0.15),
                self.settings.max_points,
            )
            elif difference>79:
                self.decision["amount"] = min(
                int(balance * 0.2),
                self.settings.max_points,
            )
            else:
                self.decision["amount"] = min(
                int(balance * (self.settings.percentage / 100)),
                self.settings.max_points,
            )
```

In simple terms the code places the 15% of the balance as a bet if the higher odds percentage is between 75 and 90, 20% of the balance when it is >90 and the default amount in other cases i.e. when lower than 75%. It was out of my capacity to just get the greater of the two values for the Odds Percentage so I achieved this using the difference of the two percentages which was used in the SMART betting strategy. 
I know I could try making a PR but I would not be able to with my limited knowledge of the code so I leave this as a suggestion for someone else to hopefully make use of or develop.
